backend/src/org/commcare/suite/model/Menu.java

```diff
@@ -78,6 +75,14 @@ public String getId() {
     }
     
     /**
+     * @return A string representing an XPath expression to determine
+     * whether or not to display this menu.
+     */
+    public String getRelevancyCondition() {
```

Comment:
This name is now pretty confusing with the one below. Maybe we should update getRelevantCondition() below to be something like "getCommandRelevantCondition()" to make it clear that they are doing different things?